Microsoft.VisualStudio.Terminal.Embeddable/ITerminalRenderer.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.Terminal
+{
+    public delegate void TerminalDataRecievedEventHandler(object sender, string data);
+
+    /// <summary>
+    /// Terminal renderer
+    /// </summary>
+    [ComImport, Guid("150B7535-03F9-41C0-9515-17ECB8199FFE")]
+
+    public interface ITerminalRenderer : ITerminal
+    {
+        /// <summary>
+        /// Gets current cols
+        /// </summary>
+        int Cols { get; }
+
+        /// <summary>
+        /// Gets current rows
+        /// </summary>
+        int Rows { get; }
+
+        /// <summary>
+        /// Gets current window cols
+        /// </summary>
+        int WindowCols { get; }
+
+        /// <summary>
+        /// Gets current window rows
+        /// </summary>
+        int WindowRows { get; }
+
+        /// <summary>
+        /// Changes the size of the terminal.
+        /// After that the terminal size won't fit the window size anymore.
+        /// </summary>
+        Task ResizeAsync(int cols, int rows, CancellationToken cancellationToken);
+
+        /// <summary>
+        /// An event that is fired when the terminal is resized and either Rows or Cols have changed. 
+        /// </summary>
+        event EventHandler TerminalResized;
```

Comment:
`TerminalResized` event is not really needed:
For auto-fit mode, WindowResized is sufficient.
For fixed mode, the client already knows the terminal size and when it changes it.

I'll remove it.